mono/mini/mini-arm.h

```diff
@@ -310,6 +310,7 @@ typedef struct MonoCompileArch {
 	int thunks_size;
 } MonoCompileArch;
 
+#define MONO_ARCH_EMULATE_FCONV_TO_U4 1
```

Comment:
This happens both on iOS/32bit and arm/linux (with `make -C mono/mini llvmaotcheck`):
```
15:18:11.2288890 Aug 27 15:18:10 iPhone5c com.xamarin.bcltests.BCL tests group 1[1228] <Notice>: Running 'test_0_fconv_u4' ...
15:18:11.2318360 Aug 27 15:18:10 iPhone5c com.xamarin.bcltests.BCL tests group 1[1228] <Notice>: test_0_fconv_u4 failed: got 1, expected 0
15:18:11.2351870 Aug 27 15:18:10 iPhone5c com.xamarin.bcltests.BCL tests group 1[1228] <Notice>: Running 'test_0_rconv_u8' ...
15:18:11.2359450 Aug 27 15:18:10 iPhone5c com.xamarin.bcltests.BCL tests group 1[1228] <Notice>: Running 'test_0_rconv_u4' ...
15:18:11.2371880 Aug 27 15:18:10 iPhone5c com.xamarin.bcltests.BCL tests group 1[1228] <Notice>: test_0_rconv_u4 failed: got 1, expected 0
```